crates/polars-time/src/windows/duration.rs

```diff
@@ -373,6 +373,22 @@ impl Duration {
         }
     }
 
+    pub fn ensure_is_constant_duration(
```

Comment:
I don't think this should be a method on `Duration` - it would make more sense as a utility function, in my opinion.